docs/design.md

```diff
@@ -109,24 +109,25 @@ every command as a top-level worker function. (`/nautobot get-device-info <devic
 rather than `/nautobot-get-device-info <device>`, `/nautobot-get-vlan-info <vlan>`, etc.) This is because:
 
 a. On platforms such as Slack, each separate slash-command must be enabled and configured separately on the server,
-   so an excessive number of distinct top-level commands will make the chatbot inconvenient to deploy.
+so an excessive number of distinct top-level commands will make the chatbot inconvenient to deploy.
 b. Platforms such as Microsoft Teams may limit the number of top-level commands that are displayed to the user in
-   a chat client, so large numbers of commands may be difficult to discover.
+a chat client, so large numbers of commands may be difficult to discover.
```

Comment:
This looks wrong to me - it is rendering as a paragraph instead of a list:

![image](https://user-images.githubusercontent.com/5603551/155735414-d31c5d15-9d38-42f5-af57-745863167b8b.png)